### PR TITLE
Update snapshots and workflows for Node 20

### PR DESCRIPTION
**Pull request by awharn**
**What It Does**
Changes workflow to use Node 18 and 20
Updates broken snapshots

**How to Test**

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [ ] added/updated automated tests
- [ ] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**

### CODE REVIEW

File: packages/imperative/src/config/cmd/report-env/EnvItems.ts [hunk 1]
```diff
@@ -69,8 +69,8 @@ function formatLogLevelMsg(logTypeName: string) {
 export const probTests: IProbTest[] = [
     {
         itemId: ItemId.NODEJS_VER,
-        probExpr: "semver.satisfies('{val}', '<14.x || 15.x || 17.x || >=19.x')",
-        probMsg: "Only Node.js versions 14, 16, and 18 are supported."
+        probExpr: "semver.satisfies('{val}', '<17.x || 19.x || >=21.x')",
```
Comment (gejohnston) [inline packages/imperative/src/config/cmd/report-env/EnvItems.ts:72]: you will find that this change has already been made in cli/add-imp-v2.

**Review by gejohnston — CHANGES_REQUESTED** [review]
Sorry, I think that we miscommunicated about where these changes can be made. I was thinking that we can merge from CLI/master into CLI/add-imp-v2. You were thinking that your changes have to placed into imperative/master because these files do not exist in the master branch of CLI. They only exist in the master branch of imperative.

We can no longer merge from imperative/master into cli/add-imp-v2 because there has been too much structural change.

I think that we have to ask you to discard this PR and create your changes in a local copy of cli/add-imp-v2 that you can then push to the remote version of cli/add-imp-v2.

**Comment by gejohnston** [comment]
It occurs to me that you still have to make these changes to imperative/master just to get the current Zowe release out the door. I approve all of the updates. I just have to ask you to make them again for the CLI/add-imp-v2 branch. Sorry that you have to do duplicated work.

**Comment by zFernand0** [comment]
@awharn 
is this good to be closed?

**Pull request closed without merging**